gimme

```diff
@@ -520,9 +520,12 @@ _list_versions() {
 	fi
 
 	local current_version
-	current_version="$(go env GOROOT 2>/dev/null)"
```

Comment:
Wouldn't it be simpler to just allow this line to fail silently as-is?  Ala:

```bash
	current_version="$(go env GOROOT 2>/dev/null || :)"
```

Reply:
If next lines are changed with code that assume `current_version` is not empty, it could introduce the bug again. Also, I always prefer to read code that explicitly states its intention, instead of relying on implicit behavior.